### PR TITLE
Initial wrapping for translations

### PR DESCRIPTION
**Pull request by davidjharder**

### CODE REVIEW

File: src/solus-mate-transition-tool.py [hunk 1]
```diff
@@ -271,14 +274,14 @@ def on_resolve_async(source, result, data=None) -> bool:
                 name = i.get_id()
                 is_installed = i.get_info() & PackageKitGlib.InfoEnum.INSTALLED == 1
                 not_installed = i.get_info() & PackageKitGlib.InfoEnum.INSTALLED == 0
-                print("is_installed", is_installed, name)
+                print(_("is_installed"), is_installed, name)
```
Comment (davidjharder) [inline src/solus-mate-transition-tool.py]: Hey @joebonrichie should the string here be "is installed"? instead?

Comment (joebonrichie) [reply src/solus-mate-transition-tool.py]: yeah looks like it

Comment (davidjharder) [reply src/solus-mate-transition-tool.py]: Changed it

**Comment by EbonJaeger** [comment]
I think we probably don't want to translate messages passed to `print()`; those would just be shown in the terminal and may be needed by us if we have to diagnose something.

**Comment by joebonrichie** [comment]
Can you resolve conflicts and i'll merge

**Comment by davidjharder** [comment]
@joebonrichie should be good to go

**Review by joebonrichie — APPROVED** [review]

**Pull request merged**